rust/debian/Dockerfile.base

```diff
@@ -69,10 +69,24 @@ RUN cd /tmp \
 ENV RUSTC_WRAPPER="/usr/local/bin/sccache"
 ENV CC_WRAPPER="/usr/local/bin/sccache"
 
+RUN cd /tmp \
```

Comment:
Tweaked the cargo audit install to use a binary build of the tool now that they support it, as we do with sccache already.